UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helpers/NMHelperTest.cs

```diff
@@ -237,6 +237,36 @@ public void WhenValidateParametersForDownloadSingleFileIsCalled_ThenShouldNotThr
             Assert.DoesNotThrow(() => NMHelper.ValidateParametersForDownloadSingleFile(parameters, string.Empty, fakeLogger));
         }
 
+        [Test]
+        public void WhenValidateParametersForDownloadDailyFileIsCalled_ThenShouldThrowException()
```

Comment:
This method name doesn't make sense.  It's the same as the method name below, except one throws an exception and one doesn't.  Do you mean "WhenInvalidParametersForDownloadDailyFileIsCalled_ThenShouldThrowException?